string weight_name
float64 distance
float64 duration
float64 weight
geometry_msgs/Point[] coordinates
RouteLeg[] legs
